tests/docker/build.sh

```diff
@@ -70,8 +72,9 @@ cargo cinstall --prefix=/usr --release
 cd
 git clone -b v0.9.1 --depth 1 https://gitlab.com/AOMediaCodec/SVT-AV1.git
 cd SVT-AV1
-cd Build/linux
-./build.sh --prefix=/usr release install
+cd Build
+cmake -G Ninja -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release ..
+ninja install
```

Comment:
I personally would recommend instead

```suggestion
cmake -B Build -S . -G Ninja -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release
cmake --build Build --target install
```
to remove the cd, but this seems good too

Reply:
Thank you for the suggestion. I suggest we first check in this version because it is similar to the other cmake and ninja commands in this file.

We can write another pull request to switch to your method throughout the file. Does this plan sound good?

Reply:
Sounds good to me